src/static/locales/en/mentors.json

```diff
@@ -10,6 +10,16 @@
     "skills": "I can also help with",
     "unavailable": "Unavailable"
   },
+  "empty": {
+    "title": "No Mentors Found",
+    "description": "No mentors were found matching your criteria.",
+    "tips": {
+      "title": "Search Tips",
+      "tip1": "Use words in a basic form in the search field, for example, \"Depression\" or \"Childhood Trauma\".",
```

Comment:
For the sake of consistency let's maybe use `<i>` tags instead of quotes with examples etc. See https://github.com/sos-lapsikyla/ylitse-web/blob/master/src/static/locales/fi/mentors.json#L23 (this is missing from `/en/mentors.json` btw :man_facepalming:).